export_snowflake/db_sync.py

```diff
@@ -543,12 +544,9 @@ def generate_temporary_external_s3_stage(self, bucket, prefix, s3_credentials):
                 self.logger.debug('Creating temporary external stage: %s', stage_generation_query)
                 self.logger.debug(self.schema_name)
                 
-                # need to point to the correct schema before creating the stage
-                default_schema_query = self.use_default_schema()
-                cur.execute(default_schema_query)
-                
-                cur.execute(stage_generation_query)
-        return
+        # temporary stage will only exist during the connection cursor session
+        # only execute the generation query later during the data loading session
```

Comment:
what does this exactly mean? does this mean that if we open connection with cursor as below, temp stage will exist until we exit that with statement block? 
```
with self.open_connection as connection:
    with connection.cursor() as cur:
```

Reply:
yea exactly, same living session as the the connection.cursor object

Reply:
Good catch, the connection object is cleaned up. log updated